tests/hw2/test_curry.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from src.hw2.curry import curry_explicit, uncurry_explicit
+
+
+def test_curry_uncurry():
+    f2 = curry_explicit((lambda x, y, z: f"<{x},{y},{z}>"), 3)
```

Comment:
Similar comment that on line 30. 
please test curry and uncurry operation separately and if testing curry/uncurry assert the equal result